### ISSUE TITLE
CheckboxGroup component

### ISSUE THREAD

**Issue by mmahalwy** [mantinedev/mantine#817]
### Usage example, including component/hook API

Would work just like `Chips` component, but checkboxes! 

```tsx
<CheckboxGroup>
  <Checkbox value="react">R label="act" />
  <Checkbox value="ng">Ang label="lar" />
  <Checkbox value="svelte">Sv label="lte" />
  <Checkbox value="vue" label="Vue" />
</CheckboxGroup>
```

### Do you want to contribute this feature and create a pull request

Yes

**Comment by rtivital** [comment]
I'll add CheckboxGroup myself in 4.0 release, it will work similar to RadioGroup

**Comment by mmahalwy** [comment]
@rtivital Sounds good to me, is there an ETA?

**Comment by rtivital** [comment]
No, it's ready when it's ready

**Comment by mmahalwy** [comment]
@rtivital ok! I can help if that helps. Can open a PR.

**Comment by rtivital** [comment]
Released in 4.0
